typescript/sdk/src/middleware/liquidity-layer/LiquidityLayerRouterDeployer.ts

```diff
@@ -49,33 +50,55 @@ export type BridgeAdapterConfig = {
 
 export type LiquidityLayerConfig = RouterConfig & BridgeAdapterConfig;
 
-export class LiquidityLayerDeployer extends MiddlewareRouterDeployer<
+export class LiquidityLayerDeployer extends ProxiedRouterDeployer<
   LiquidityLayerConfig,
   LiquidityLayerFactories,
-  LiquidityLayerRouter__factory
+  'liquidityLayerRouter'
 > {
   readonly routerContractName = 'liquidityLayerRouter';
 
-  constructor(
-    multiProvider: MultiProvider,
-    configMap: ChainMap<LiquidityLayerConfig>,
-    create2salt = 'LiquidityLayerDeployerSalt',
-  ) {
-    super(multiProvider, configMap, liquidityLayerFactories, create2salt);
+  constructor(multiProvider: MultiProvider) {
+    super(multiProvider, liquidityLayerFactories);
+  }
+
+  async constructorArgs(_: string, __: LiquidityLayerConfig): Promise<[]> {
+    return [];
+  }
+
+  async initializeArgs(
```

Comment:
Why aren't we pulling this from ProxiedRouterDeployer?

Reply:
technically the `initialize` could look different for the liquidity layer